src/Components/Aspire.Azure.Messaging.WebPubSub/AspireWebPubSubExtensions.cs

```diff
@@ -74,6 +74,29 @@ public static void AddKeyedAzureWebPubSubServiceClient(
         new WebPubSubComponent().AddClient(builder, DefaultConfigSectionName, configureWithServiceKeyAsDefaultHubName, configureClientBuilder, connectionName: connectionName, serviceKey: serviceKey);
     }
 
+    /// <summary>
+    /// Registers <see cref="WebPubSubServiceClient"/> as a singleton for given <paramref name="connectionName"/> in the services provided by the <paramref name="builder"/>. This
+    /// overload does not require a service key and uses the connection name as the service key
+    /// to support scenarios where multiple Hubs are referenced in the same application.
```

Comment:
Should we document that we expect the `connectionName` to be the name of a `Hub` resource?

Reply:
The `connectionName` doesn't end up being the name of the hub resource, just the service key. We derive the name of the hub resource from the connection string denoted by `connectionName` though.